Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -1630,7 +1612,10 @@ void RenderStyle::applyMotionPathTransform(TransformationMatrix& transform, cons
         anchor = floatPointForLengthPoint(offsetAnchor(), boundingBox.size()) + boundingBox.location();
     
     // Shift element to the point on path specified by offset-path and offset-distance.
-    auto path = getPathFromPathOperation(boundingBox, *offsetPath(), anchor, offsetRotate());
+    PathOperation* offset = offsetPath();
+    offset->setTransformInfo(boundingBox, anchor, offsetRotate());
```

Comment:
It's not OK to mutate the PathOperation in the RenderStyle (because bits of RenderStyle can be shared between renderers).

Reply:
Ah I see. I made them function parameters with default values now.